packages/eslint-plugin-template/tests/rules/i18n/cases.ts

```diff
@@ -46,13 +46,16 @@ export const valid = [
     `,
     options: [{ ignoreTags: ['mat-icon'] }],
   },
+  `<div>-{{data_from_backend}}</div>`,
+  `<div>1{{data_from_backend}}</div>`,
+  `<div>-1{{data_from_backend}}</div>`,
   {
     code: `
-      <div i18n-tooltip="@@tooltip.label" tooltip="This tooltip property is ignored">
-        -{{data_from_backend}}
+      <div>
+        My company untranslatable name{{data_from_backend}}
```

Comment:
Thinking more about it, I think we still should address #412, because one may still needs to exclude some texts from `Text` (not only from `BoundText`).

Reply:
Sounds good, thanks! I'll create a v13 branch soon as well